recipes/cppcheck/all/conanfile.py

```diff
@@ -14,37 +13,30 @@ class CppcheckConan(ConanFile):
     topics = ("code quality", "static analyzer", "linter")
     description = "Cppcheck is an analysis tool for C/C++ code."
     license = "GPL-3.0-or-later"
+    package_type = "application"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"with_z3": [True, False], "have_rules": [True, False]}
```

Comment:
We like to deprecated options before removing them so users have a chance 
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-a-recipe

Reply:
I set the option to deprecated as suggested in the FAQ document